vmngclient/api/parcel_api.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from vmngclient.session import vManageSession
+
+from vmngclient.endpoints.configuration_feature_profile import ParcelId, SDRoutingConfigurationFeatureProfile
+from vmngclient.model.feature_profile_parcel import FullConfig, FullConfigParcel
+
+
+class ParcelAPI(ABC):
```

Comment:
Considered using `typing.Protocol` instead `ABC`? I find it more suitable for that use case.